stories/victory-charts/victory-animation/config.ts

```diff
@@ -0,0 +1,21 @@
+import type { Meta, StoryObj } from "@storybook/react";
+
+import { VictoryAnimation } from "@/victory";
+import { componentContainer } from "../../utils/decorators";
+
+type StoryProps = React.ComponentProps<typeof VictoryAnimation> & {
+  themeKey: string;
+};
+
+export const ComponentMeta: Meta<StoryProps> = {
+  component: VictoryAnimation,
+  decorators: [componentContainer],
+
+  argTypes: {
+    delay: { control: "number" },
+    duration: { control: "number" },
+    easing: { control: "string" },
```

Comment:
Been noticing type errors around `control: "string"`. Looking at the storybook docs, it looks like it should be `control: "text"` instead?